README.md

```diff
@@ -308,8 +309,8 @@ This application defines Docker images for production, testing, and development
 | Technology | End of Support | Notes | Affected Files in ci/ |
 |------------|----------------|-------|-----------------------|
 | Python 3.8 | 14 October 2024 | | Dockerfile, Dockerfile.local, Dockerfile.test, Dockerfile.userflow |
-| Alpine Linux 3.15 | 1 November 2023 | | Dockerfile, Dockerfile.local, Dockerfile.test |
-| Postgres 11.8 | 9 November 2023 for v11.x | The YAML files specify v11.8.  As of March 2022, versions 11.15 and 14.2 are available. | docker-compose.yml, docker-compose-local.yml, docker-compose-test.yml |
+| Alpine Linux 3.15 | 1 November 2023 | | Dockerfile, Dockerfile.local, Dockerfile.test, Dockerfile.db |
```

Comment:
I removed the reference to Dockerfile.db from an earlier revision.  I'm not pushing code to the branch again for that.